mk/image.lds.S

```diff
@@ -92,15 +94,23 @@ SECTIONS {
 
 		ALIGNMENT();
 		_reserve_end = .;
+		_bss_reserve = _reserve_end - _bss_end;
```

Comment:
The same is for this. Once you have _bss_vma, _bss_len, and _bss_len_with_reserve, you have no need to define something else, because it would be redundant. You can derive end address and length of both bss and bss_with_reserve in the C code just using these three.

Reply:
Fixed